Updating user doc (#234)
DataDog/KubeHound: deployments/kubehound/README.md

# KubeHound docker stack

> KubeHound is running under the hood dockers for the storedb, graphdb and UI.

Reviewer: ```suggestion
Under the hood, KubeHound is running dockers for the storedb, graphdb and UI.
```
sounds a bit better to me

Author: fixed

## KubeHound backend

> The docker backend can be handled directly from `kubehound` binary. To see the commands please refer to [Backend](https://kubehound.io/user-guide/common-operations/#backend).

Reviewer: ```suggestion
The docker backend can be handled directly with the `kubehound` binary. Check the [Backend](https://kubehound.io/user-guide/common-operations/#backend) documentation.
```
